docs/docs/docs_es/api-reference/core/selector.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: selector(options)
+título: selector(options)
```

Comment:
Don’t change the frontmatter keys, these are internal.

Reply:
Reversed change. Thanks!